lib/rmt/cli/mirror.rb

```diff
@@ -1,29 +0,0 @@
-class RMT::CLI::Mirror < RMT::CLI::Base
-
-  default_task :repos
-
-  desc 'repos', 'Mirror enabled repositories'
-  def repos
-    repos = Repository.where(mirroring_enabled: true)
-    if repos.empty?
-      warn 'There are no repositories marked for mirroring.'
-      return
-    end
-    repos.each { |repo| mirror(repo) }
-  end
-
-  desc 'custom URL', 'Mirror a custom repository URL'
-  def custom(url, path = nil)
-    url += '/' unless url.end_with?('/')
```

Comment:
Was there a reason to remove this functionality?

It could still be useful to mirror things like openSUSE repositories until we also can also get custom products in.